pkg/docker/docker_api.go

```diff
@@ -0,0 +1,283 @@
+package docker_api
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"net/url"
+	"os"
+	"os/user"
+	"path/filepath"
+	"time"
+
+	"github.com/appscode/analytics/pkg/spreadsheet"
+	"github.com/appscode/errors"
+	"golang.org/x/net/context"
+	"golang.org/x/oauth2/google"
+	"google.golang.org/api/sheets/v4"
+)
+
+type DockerRepositories struct {
+	Count    int         `json:"count"`
+	Next     interface{} `json:"next"`
+	Previous interface{} `json:"previous"`
+	Results  []struct {
+		User           string      `json:"user"`
+		Name           string      `json:"name"`
+		Namespace      string      `json:"namespace"`
+		RepositoryType string      `json:"repository_type"`
+		Status         int         `json:"status"`
+		Description    string      `json:"description"`
+		IsPrivate      bool        `json:"is_private"`
+		IsAutomated    bool        `json:"is_automated"`
+		CanEdit        bool        `json:"can_edit"`
+		StarCount      int         `json:"star_count"`
+		PullCount      int         `json:"pull_count"`
+		LastUpdated    time.Time   `json:"last_updated"`
+		BuildOnCloud   interface{} `json:"build_on_cloud"`
+	} `json:"results"`
+}
+
+type DockerRepoLogs struct {
+	User        string
+	Name        string
+	StarCount   int
+	PullCount   int
+	LastUpdated time.Time
+}
+
+const SpreadSheetIdKubeDB = "10OGrTJxEDox4VR15U7HPGRjiFpTNfhiUGMuL7BQJKJU"   //https://docs.google.com/spreadsheets/d/10OGrTJxEDox4VR15U7HPGRjiFpTNfhiUGMuL7BQJKJU
+const SpreadSheetIdAppsCode = "18lNbYqiP4gsKBoLDoUw2ejOGWN3t3mUOyKGaKeye3kI" //https://docs.google.com/spreadsheets/d/18lNbYqiP4gsKBoLDoUw2ejOGWN3t3mUOyKGaKeye3kI
+
```

Comment:
If we are going to stay with google docs now, at least make them as flags or EnvVar